docs/modules/reference/pages/manage/app/upload.adoc

```diff
@@ -1,5 +1,49 @@
 = Uploading Applications
-// https://manage.payara.cloud/application/8744ddec-d70d-45ef-8aa8-0f6f6f7bafff/n/start:dev:ed5cb015/upload/
-// upload dialog
-// inspection screen
-// payara-cloud.properties
\ No newline at end of file
+
+Applications in Payara Cloud are uploaded to Namespaces.
+Different applications can be uploaded to the same Namespace.
+
+.Options to upload new application in a Namespace
+image::manage/application/application-9-upload-namespace.png[]
+
+[checklist]
+* ❶ Namespace: The Namespace within which the application will be uploaded
+* ❷ Namespace Actions Menu: Menu option to upload new application
+* ❸ Upload Button: Dedicated new application upload button
+
+Clicking either of the new application upload options above brings up the following upload diaglog.
+
+.New application upload popup dialog
+image::manage/application/application-10-upload.png[]
+
+[checklist]
+* ❶ Binary Upload Button: Click to bring up the file upload dialog to pick an application binary (.WAR file).
+* ❷ Name Of Artefact: Shows the name of the selected application binary
+* ❸ Deploy Option: Option to directly deploy the uploaded binary
+* ❹ Application Name: Option to give the application a name distinct from the uploaded binary file
+
+An uploaded application as shown in the above dialog will transition to the xref:manage/app/detail.adoc#_pending_state[Pending State].
+Custom xref:manage/app/configuration/overview.adoc[configurations] can be carried out on the uploaded application to ready it for deployment.
+You can directly deploy the application right after uploading by clicking the Deploy Immediately option in ❸.
+A directly deployed application transitions to the xref:manage/app/detail.adoc#_running_state[Running State]
+
+== Deploying Pending Applications
+
+A pending application can be deployed by clicking the Deploy Changes menu option from the Application Actions drop-down menu
```

Comment:
A pending application can be deployed by clicking the 'Deploy Changes' menu option from the Application Actions drop-down menu